src/Controls/src/Core/Device.cs

```diff
@@ -19,15 +19,16 @@ public static class Device
 		/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="//Member[@MemberName='Android']/Docs" />
 		public const string Android = "Android";
 		/// <include file="../../docs/Microsoft.Maui.Controls/Device.xml" path="//Member[@MemberName='UWP']/Docs" />
-		public const string UWP = "UWP";
+		internal const string UWP = "UWP";
```

Comment:
Should we mark them as internal or directly remove?

Reply:
I think UWP is a special case that the underlying architecture changed so this maybe should be obsolete.